default/stringtables/en.txt

```diff
@@ -9899,6 +9985,50 @@ While transforming an entire population into beings of pure energy would be impr
 
 The ultimate stage of physical development is to transcend the physical altogether. Without bodies, individuals and society may move, grow and exist freely, unhindered by unnecessary physical limitations and inefficient metabolic processes that waste energy supporting homeostasis.'''
 
+CON_TRANS_DESIGN
+Transcendent Design
+
+CON_TRANS_DESIGN_DESC
+'''Transcending the limitations of the designing species, the most creative and effective from the different species in the empire start to work together to build universally usable goods.
+Building [[building BLD_INTERSPECIES_ACADEMY]] on different planets ensures the practice spreads in the empire.
+
+The imperial stockpile service uses the outcome to prepare goods which can be used by any citizen.'''
+
+PRO_IMPERIAL_STOCKPILE_1
+Predictive Stockpiling
+
+IMPERIAL_STOCKPILE_1_DESC
+'''Knowing where demand for certain goods will be in future allows to stack the items up in time.
+The imperial stockpile service has a fleet for distributing raw materials to places where important projects are predicted to be started.
+This task is difficult as the service has to predict when demand happens, where it happens, what materials and goods are demanded and also who the users will be.
+
+This is the basic technology for the imperial stockpile.
+The imperial stockpile service starts with 20% [[metertype METER_IMPERIAL_PP_TRANSFER_EFFICIENCY]] and to draw 2PP/turn from the stockpile (See [[metertype METER_IMPERIAL_PP_USE_LIMIT]]).'''
+
+PRO_IMPERIAL_STOCKPILE_2
+Generic Supplies
+
+IMPERIAL_STOCKPILE_2_DESC
+'''By carefully choosing supply chains, materials, and designs, Generic Supplies are created which can easily be decomposited and reused for most industrial products.
+
+Generic Supplies can be created a long time before the demand occurs. Supply packages usually include bootstrap devices which reconstruct the facilities for reconstructing new products from the Generic Supplies.
+Reaching this state of sophistication, Generic Supplies can be used to construct new technology products which were not available at the time the supplies were produced.
+
+This frees the imperial stockpile service from having to know which goods will be demanded exactly. Predicting the location is enough.
+
+This is an upgrade technology for the imperial stockpile.
+It adds 20% to the [[metertype METER_IMPERIAL_PP_TRANSFER_EFFICIENCY]] and allows to draw 3PP/turn extra from the stockpile (See [[metertype METER_IMPERIAL_PP_USE_LIMIT]]).'''
+
+PRO_IMPERIAL_STOCKPILE_3
+Interstellar Entanglement Factory
+
+IMPERIAL_STOCKPILE_3_DESC
+'''Quantum entanglement of stockpiled goods allows specialists to operate machines remotely from other star systems.
+Putting the empires best engineers directly in charge of building using the stockpiled goods helps making the most out of these.
```

Comment:
empire's

whole paragraph of flavour text isn't great

Reply:
You mean the idea is not great or the writing? Or both?

Reply:
The idea is that you build a control room and its clone out of entangled particles then you ship one to the remote factory location and hook it up there. What happens to one happens to the other ;)

Reply:
References to "specialists" or "empire[']s best engineers" not good style